catalog-info.yaml

```diff
@@ -0,0 +1,24 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: "Component"
+metadata:
+  name: 'event-routing-backends'
+  description: "A package for Open edX events tracking"
+  annotations:
+    # (Optional) Annotation keys and values can be whatever you want.
+    # We use it in Open edX repos to have a comma-separated list of GitHub user
+    # names that might be interested in changes to the architecture of this
+    # component.
+    openedx.org/arch-interest-groups: ""
+spec:
+
+  # (Required) This can be a group(`group:<group_name>` or a user(`user:<github_username>`)
+  owner: "group:committers-event-routing-backends"
```

Comment:
@bmtcril is this accurate or should it be some other group?

Reply:
Hmm, was that by default?  I think that potentially this should be Edly.  Thoughts?

Reply:
Actually i found from this reference that `committers-event-routing-backends` has this repo.
<img width="1414" alt="Screenshot 2023-09-06 at 11 21 41 PM" src="https://github.com/openedx/event-routing-backends/assets/4454296/5bc93718-bbf9-4eab-9104-7cd1935ecf95">

Reply:
I'm not sure who is supposed to own it, but the actual maintainers and admins on the repo are currently these:

<img width="852" alt="Screenshot 2023-09-07 at 9 27 14 AM" src="https://github.com/openedx/event-routing-backends/assets/112640379/5e45308d-9363-4730-8a71-a7fae2858dc8">

Reply:
@bmtcril should i updated the owner to @openedx/arbi-bom?

Reply:
It sounds like @ziafazal is going to be the / a official maintainer. Zia, do you want this set to yourself or to a group?

Reply:
@salman2013 yes please set it to edly
```
       team_reviewers: 'edly'
       email_address: 'admin@edly.io'
```